src/monopoly/view/monopoly_view.js

```diff
@@ -963,7 +1065,23 @@ MonopolyView.prototype.updateProperties = function() {
             tile.addChild(ownerTag);
         }
     }
-}
+};
+
+MonopolyView.prototype.updateTradeInfo = function(){
+    
+    if(this.turnMap.getCurrentState() == PROPOSE_TRADE){
+        if(!this.game.trade){
+            this.game.createTrade();
```

Comment:
You should create the trade in the _onEnter in the state machine and not in the view